src/borg/testsuite/key.py

```diff
@@ -9,7 +10,12 @@
 from ..helpers import Location
 from ..helpers import Chunk
 from ..helpers import IntegrityError
+from ..helpers import get_key_nonces_dir
 from ..key import PlaintextKey, PassphraseKey, KeyfileKey, Passphrase, PasswordRetriesExceeded, bin_to_hex
+from ..key import NonceManager
+from ..remote import InvalidRPCMethod
+
+import borg.key  # for monkey patching NONCE_SPACE_RESERVATION
```

Comment:
For some reason i had problems with relative imports. But your suggesting works.
We could use

    from .. import key as key_module

Importing it as key would be quite confusing in the tests.